SUMMARY.md

```diff
@@ -1,24 +1,28 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/INSTALLATION.md)
+  * [Docker Installation](installation/INSTALLATION-DOCKER.md)
+  * [Local Installation](installation/INSTALLATION-LOCAL.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](neo4j/README.md)
   * [GraphQL](backend/graphql.md)
+  * [Middleware](backend/middleware.md)
+  * [Legacy Migration](backend/db-migration-worker/README.md)
 * [Webapp](webapp/README.md)
+  * [Styleguide](webapp/styleguide.md)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-24T13:27:58Z" title="Wednesday, April 24th 2019, 3:27:58 pm +02:00">Apr 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](backend/neo4j-database.md)
   * [GraphQL](backend/graphql.md)
-  * [Legacy Migration](backend/db-migration-worker/README.md)
+  * [Testing](backend/testing.md)
+  * [Middleware](backend/middleware.md)
+  * [Legacy Migration](backend/db-migration-worker.md)
```


Same, legacy migration is completely irrelevant for open source contributors. It's being moved into folder `deployment` in https://github.com/Human-Connection/Human-Connection/pull/485